packages/blog-starter-kit/themes/enterprise/components/footer.tsx

```diff
@@ -20,7 +20,7 @@ export const Footer = () => {
 						</Link>
 					</div>
 				) : (
-					<p className="mb-20 text-center text-xl font-semibold text-white md:text-4xl">
+					<p className="mb-20 text-center text-xl font-semibold text-slate-900 dark:text-white md:text-4xl">
```

Comment:
Could you please change this to `dark:text-slate-50`?

Reply:
Alright

Reply:
I have applied the correction you made

> Could you please change this to `dark:text-slate-50`?